src/remote-config/remote-config.ts

```diff
@@ -116,6 +118,28 @@ export class RemoteConfig implements FirebaseServiceInterface {
       });
   }
 
+  /**
+  * Gets a list of Remote Config template versions that have been published, sorted in reverse 
+  * chronological order. Only the last 300 versions are stored.
+  * All versions that correspond to non-active Remote Config templates (i.e., all except the 
+  * template that is being fetched by clients) are also deleted if they are older than 90 days.
+  * 
+  * @param {ListVersionsOptions} options Optional options object for getting a list of versions:
+  *    - {number} `pageSize` The maximum number of items to return per page.
```

Comment:
This kind of documentation is hard to maintain. Only indicating the type and a summary for the enclosing type is enough.